backend/src/integrations/UserProfileDAO.ts

```diff
@@ -41,18 +74,49 @@ class UserProfileDAO {
     );
   }
 
-  addUser(user: IUserProfile) {
+  /**
+   * ---------------------------------------------------------------------------
+   * Method
+   * ---------------------------------------------------------------------------
+   */
+  public addUser(user: IUserProfile) {
+    this.users.push(user);
+    this.persistUsers();
+  }
+
+  /**
+   * ---------------------------------------------------------------------------
+   * Method
+   * ---------------------------------------------------------------------------
+   */
+  public updateUser(user: IUserProfile) {
     var usersWithoutUser = this.users.filter(u => u.id !== user.id);
     usersWithoutUser.push(user);
     this.users = usersWithoutUser;
     this.persistUsers();
   }
 
-  deleteUser(user: IUserProfile) {
+  /**
+   * ---------------------------------------------------------------------------
+   * Method
+   * ---------------------------------------------------------------------------
+   */
+  public deleteUser(user: IUserProfile) {
```

Comment:
Do we need to methods for deleting a user? Looks like this one is not used. yet

Reply:
used in the api endpoint [DELETE] `/user/:userid`

Reply:
haven't scanned the code for use of this one, since it already existed.